Upstream-status: Backport

The warnings below are actual errors when building.

Signed-off-by: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
---
From 99c69dc3ca363ce9ff909531f8581252189905d0 Mon Sep 17 00:00:00 2001
From: "John M. Schanck" <jschanck@mozilla.com>
Date: Mon, 13 Dec 2021 16:35:17 +0000
Subject: [PATCH] Bug 1738028: avoid a clang 13 unused variable warning in opt
 build. r=bbeurdouche

Differential Revision: https://phabricator.services.mozilla.com/D130309

--HG--
extra : moz-landing-system : lando
---
 lib/certhigh/certvfypkix.c | 2 ++
 lib/freebl/shvfy.c         | 1 +
 2 files changed, 3 insertions(+)

diff --git a/nss/lib/certhigh/certvfypkix.c b/nss/lib/certhigh/certvfypkix.c
index 95d88b53f..7c8784f4e 100644
--- a/nss/lib/certhigh/certvfypkix.c
+++ b/nss/lib/certhigh/certvfypkix.c
@@ -817,6 +817,8 @@ cert_PkixErrorToNssCode(
     void *plContext)
 {
     int errLevel = 0;
+    (void)errLevel; /* Suppress unused var warning (Bug 1738028) */
+
     PKIX_Int32 nssErr = 0;
     PKIX_Error *errPtr = error;
 
diff --git a/nss/lib/freebl/shvfy.c b/nss/lib/freebl/shvfy.c
index da42467fd..0428bf6c0 100644
--- a/nss/lib/freebl/shvfy.c
+++ b/nss/lib/freebl/shvfy.c
@@ -335,6 +335,7 @@ blapi_SHVerifyFile(const char *shName, PRBool self)
     SECStatus rv;
     DSAPublicKey key;
     int count;
+    (void)count; /* Suppress unused var warning (Bug 1738028) */
 #ifdef FREEBL_USE_PRELINK
     int pid = 0;
 #endif
-- 
2.41.0

